common/include/pcl/common/polynomial_calculations.h

```diff
@@ -49,15 +49,11 @@ namespace pcl
   class PolynomialCalculationsT 
   {
     public:
-      // =====CONSTRUCTOR & DESTRUCTOR=====
-      PolynomialCalculationsT ();
-      ~PolynomialCalculationsT ();
-      
       // =====PUBLIC STRUCTS=====
       //! Parameters used in this class
       struct Parameters
       {
-        Parameters () : zero_value (), sqr_zero_value () { setZeroValue (1e-6);}
+        Parameters () { setZeroValue (1e-6);}
```

Comment:
If `real` is int then the variables will go initialized. Maybe default init them with `zero_value = {}` or similar